web/static/js/socket.js

```diff
@@ -1,19 +1,18 @@
 // NOTE: The contents of this file will only be executed if
-// you uncomment its entry in "web/static/js/app.js".
+// you uncomment its entry in 'web/static/js/app.js'.
```

Comment:
It was making my linter happy (semistandard) which uses single '. It's done in the comments too because I just did `ctrl+d` on all the ".